include/circt/Dialect/ESI/ESIOps.h

```diff
@@ -10,6 +10,7 @@
 #include "circt/Dialect/ESI/ESIDialect.h"
 #include "circt/Dialect/ESI/ESITypes.h"
 
+#include "mlir/IR/DialectImplementation.h"
```

Comment:
Is the header needed here, or could it just be added in ESIOps.cpp?

Reply:
OpImplementation.h is what's actually required here.